storybookjs/storybook
# Fix dynamic knobs

Issue: #2661 accidentally broke "dynamic knobs" feature (https://github.com/storybook-eol/storybook-addon-knobs/pull/59)

## How to test

Open [dynamic knobs story](https://5adb800d3672df3a2f445d2e--storybooks-official.netlify.com/?knob-Show%20optional=yes&knob-compulsary=I%20must%20be%20here&knob-optional=I%20can%20disapear&selectedKind=Addons%7CKnobs.withKnobs&selectedStory=dynamic%20knobs&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel), select "no". Optionsl knob should disappear
